api/pom.xml

```diff
@@ -44,6 +44,10 @@
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+		</dependency>
```

Comment:
Do we explicitly need this here? I kind of preferred implicitly loading it via HAPI.